src/lib.rs

```diff
@@ -652,6 +654,10 @@ pub enum SymbolSrcCfg {
     ///
     /// With a `None` value, it would means a process calling BlazeSym.
     Process { pid: Option<u32> },
+    Gsym {
+        file_name: String,
```

Comment:
Should this be a `PathBuf`?

Reply:
Well, it's the correct thing to use. It also provides native support for path manipulations. String can certainly be partly stronger in its assurances (it's guaranteed to be UTF-8), so the library may not be able to work correctly on certain file systems with certain paths, because they may not be representable as a string. And unless we can iterate those potential incompatibility pitfalls from the top of our heads and without an hour of research we are probably well advised to just use the intended thing for its intended purpose.

Reply:
Ok! UTF-8 is a potential issue.  I just updated the code to use PathBuf.  Path is unsized, so I used PathBuf instead.